pkg/minishift/provisioner/minishift_provisioner.go

```diff
@@ -147,11 +152,9 @@ func (provisioner *MinishiftProvisioner) GenerateDockerOptions(dockerPort int) (
 
 	// systemd / redhat will not load options if they are on newlines
 	// instead, it just continues with a different set of options; yeah...
-	engineConfigTemplate := ""
+	engineConfigTemplate := engineConfigTemplateCentOS
```

Comment:
I had actually more improvements for this code, but postponed them for now. Think I can share them and let's discuss...